app/src/main/res/values/theme_mutedpurple.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <color name="theme_muted_purple_accent">#627EC8</color>
+    <color name="light_grey_transparent">#2EAAAAAA</color>
+
+    <style name="Theme.Jellyfin.MutedPurple" parent="Theme.Jellyfin">
+        <!-- Colors -->
+        <item name="android:colorAccent">@color/theme_muted_purple_accent</item>
+        <item name="defaultSearchColor">@color/theme_muted_purple_accent</item>
+        <item name="progressPrimary">@color/theme_muted_purple_accent</item>
+        <item name="android:colorButtonNormal">@color/light_grey_transparent</item>
+
+        <!-- Rounding Values -->
+        <item name="cardRounding">5dp</item>
+        <item name="buttonRounding">7dp</item>
```

Comment:
I think this might be a bit too much rounding:

![image](https://user-images.githubusercontent.com/2305178/117002334-b807ba80-ace3-11eb-9dfa-3b7b7aa20e42.png)

Reply:
Looked at it again and I actually don't mind it now :p

Reply:
Ah cool :) I've been testing 5dp and 6dp on my tv the past couple days and I do still prefer 7dp